### ISSUE TITLE
Display element not accessible by screen reader

### ISSUE THREAD

**Issue by bryce** [bryce/onlinecalculatortest#8]
The Display element has no aria tag nor any other tag/value that is accessible by screen reader
ie)
<input type=""button"" class=""window"" name=""display"" id=""display"" value="""">

It needs one for a screen reader to pick it up and correctly identity it

Severity: S2

Reference: TC75
